src/standard.js

```diff
@@ -1,11 +1,35 @@
-export default function standard(cost) {
+// emoji, attack, health, name, text, onSummon
+export function standard(cost) {
   const emoji = emojis[cost] || emojis[emojis.length - 1];
   return {
     cost: cost,
     emoji: emoji,
     attack: cost,
     health: cost,
-    name: `Standard ${cost}-${cost}`
+    name: `Standard ${cost}-${cost}`,
+    text: ""
+  };
+}
+//need action to change health or attack of a creature already made by game
+export function battlecry(cost) {
+  const emoji = emojis[cost] || emojis[emojis.length - 1];
+  return {
+    cost: cost,
+    // emoji: emoji,
+    attack: cost,
+    health: cost,
+    name: "Sunkeeper Tarim",
```

Comment:
let's try and avoid putting any trademarked names in this. feel free to come up with something else our put something generic